src/core/containers/HandleLogin/index.js

```diff
@@ -58,7 +59,14 @@ function createLoadData(dispatch) {
             secure: config.get('cookieSecure'),
             maxAge: config.get('cookieMaxAge'),
           });
-          router.push('/search');
+          let to;
+          try {
+            to = browserBase64Decode(state.split(':')[1]);
```

Comment:
with cases like that I just worry about some new code that forgets to update an old index if the state elements needs to change but no biggie